app/controllers/participants/validations_controller.rb

```diff
@@ -254,7 +254,7 @@ def store_form_and_redirect_to_step(step)
     end
 
     def store_analytics(matched:)
-      Analytics::ECFValidationService.record_validation(
+      Analytics::RecordValidationJob.perform_later(
```

Comment:
Does this work? I didn't think sidekiq jobs could take AR models? https://github.com/mperham/sidekiq/wiki/The-Basics#client

Reply:
https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple

Reply:
Yes it does https://github.com/mperham/sidekiq/wiki/Active-Job#using-global-id

We could change all the jobs to find by ID instead of using the active job functionality, but maybe that is more of a tech debt task as it would make this PR even larger